source/Playnite/GamesEditor.cs

```diff
@@ -260,7 +260,19 @@ public void OpenGameLocation(Game game)
 
             try
             {
-                Process.Start(game.ExpandVariables(game.InstallDirectory));
+                var gameClone = game.GetClone();

```

Comment:
Correct if I'm wrong, but I don't see a reason to create game copy here.

Reply:
I am creating a copy, because I change the provided Game object. But as you mention it now, i just noticed that InstallDirectory can contain placeholder and so i need to resolve these placeholders before I can check if the path or alternatives exist

Reply:
Just store original install directory into a variable and work with that, making copy of the game object is unnecessary.

Reply:
But first I need to expand the placeholders before I check for alternatives, don't I?

Reply:
But I think it makes actually no sense using placeholders for install direcotry because it is used to resolve placeholder {InstallDir}, which could lead to the problem that {InstallDir} is resolved with the value {InstallDir}, or am I wrong? ;-)

Reply:
I don't know why there's expanding going on in the first place. I think we can remove that, because it IMO doesn't make sense to have variables in installation directory (I can't think of a use case for them there). I don't remember why I added expanding here.